### DISCUSSION TITLE
Discussion Icons Enlarged

### DISCUSSION THREAD

**Discussion by JustinBack** [community/community#3254] [Mobile] [answered]
Just noticed this on iOS today version 1.4

![image](https://user-images.githubusercontent.com/40178814/106180051-1cbef080-619c-11eb-925d-f5aa88e5d924.jpeg)

That some icons (custom set) are enlarged

![image](https://user-images.githubusercontent.com/40178814/106180136-306a5700-619c-11eb-9362-f163c66363e9.jpeg)

**Comment by candyho** [comment, marked as answer]
Hey, thanks for reporting this bug. This should be fixed in the latest build that was rolled out yesterday - update the app and try again!
